docs/examples/README.md

```diff
@@ -0,0 +1,12 @@
+Examples
+---
+
+***Not all of the examples are meant to work without modification.***
+
+The version suffix is only meant to indicate an alternate version
+of the same example. e.g. `custom-headers-v2.js` is meant to be used
+with the latest version of Cloudscraper, ***not*** Cloudscraper v2.0.0.
+
+If you've noticed for some reason that an example is outdated and/or misleading, please send a PR to correct it.
+<br>In fact, updating or adding examples is a great way to contribute!
```

Comment:
Do we need `<br>` in markdown?

Reply:
The line breaks made the formatting nicer. You can play with that to see what I mean. There is no reason not to have them in Github flavored markdown. It's supported.

Reply:
You'll have to look at the source of the markdown documents that I am linking because comments on Github work differently. I don't know if you've noticed but I always have extra spaces in my comments because I always just do it one way.

The lines are wrapped if they are newline delimited: https://gist.github.com/pro-src/e72c4c1724df4a1fd6734bbb2c6f7dbc

The lines are not wrapped when followed by two newlines but have an empty line between them: https://gist.github.com/pro-src/926f67f256d329a7b301a46087a789db

The lines are not wrapped when using the supported `<br>` tag: https://gist.github.com/pro-src/9661ec95cb189c0fa04b36521fdff650

The spec: https://github.github.com/gfm/#raw-html

So the `<br>` is used to add line breaks without extra new lines.

Reply:
And yet again thank you for the explanation :)

Reply:
You're welcome. :smiley: